mllib-dal/src/main/scala/com/intel/oap/mllib/feature/PCADALImpl.scala

```diff
@@ -24,9 +26,8 @@ import org.apache.spark.annotation.Since
 import org.apache.spark.internal.Logging
 import org.apache.spark.ml.linalg._
 import org.apache.spark.mllib.feature.{PCAModel => MLlibPCAModel, StandardScaler => MLlibStandardScaler}
-import org.apache.spark.mllib.linalg.{DenseVector => OldDenseVector, DenseMatrix => OldDenseMatrix, Vectors => OldVectors}
+import org.apache.spark.mllib.linalg.{DenseMatrix => OldDenseMatrix, DenseVector => OldDenseVector, Vectors => OldVectors}
 import org.apache.spark.rdd.RDD
-
 import java.util.Arrays
 
 class PCADALModel private[mllib] (
```

Comment:
pls check if it is necessary to normalizeData for the following line:

```
val normalizedData = normalizeData(data)
```

There is no normalization in vanilla Spark when apply correlation-based PCA.
Check comments of: 
https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/RowMatrix.scala#L469

You need to add both normalized data and unnormalized data in unit tests.